### ISSUE TITLE
Add TASA Typeface Collection

### ISSUE THREAD

**Issue by eliheuer** [google/fonts#6605]
**Font Project Git Repo URL:** https://github.com/adrianzwz/TASA-Typeface-Collection

**Super short description of the Font Family:**

https://tasatype.localremote.co/
https://localremote.co/work/187b16cf-06f8-4a0a-9560-95d0811c4709

The TASA Typeface Collection consists of TASA Explorer and TASA Orbiter, two bespoke typefaces designed for the rebrand of Taiwan Space Agency. They shape the typographical voice of the institute for communications across territorial boundaries, marking a milestone in its history and signaling what future awaits.

**Requirements:**

Google Fonts will publish only fonts that match its requirements. Please familiarize yourself with the complete documentation in the [Google Fonts Guide](https://googlefonts.github.io/gf-guide/) (GF-Guide) and ensure your font project complies with them before submitting the font family. You can also use the [Google Fonts Project Template](https://github.com/googlefonts/googlefonts-project-template), which will help you create a repository that follows the needed structure and includes build requirements.

By filling this issue, you can confirm the project meets the  requirements (by ticking the cases or putting x between the square brackets in text mode):

- [ ] I read, agree, and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/index#pre-production-getting-your-fonts-ready-for-gf)
- [x] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [ ] There is no "Reserved Font Name" mention in the OFL license
- [x] Family name is unique according to [namecheck.fontdata.com](https://namecheck.fontdata.com/)
- [ ] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
- [ ] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
- [x] The source files are available in the repository
- [ ] I am the author of the font, or the author agreed to this submission
- [ ] I will maintain the repository and participate in the onboarding process (addressing, solving, and responding to issues, merging pull requests, etc)

**Image:**

<img width="901" alt="Screenshot 2023-08-08 at 12 18 25 PM" src="https://github.com/google/fonts/assets/5162664/dfeb3ff6-4160-4611-b30e-d2c910a3abf7">

<img width="1042" alt="Screenshot 2023-08-08 at 12 18 56 PM" src="https://github.com/google/fonts/assets/5162664/1f759b9b-dd44-4d13-8fdf-ca8eac60302a">

<img width="1113" alt="Screenshot 2023-08-08 at 12 19 20 PM" src="https://github.com/google/fonts/assets/5162664/09d66ae5-e6f3-4f1a-ae54-ddba4dd3256d">

<img width="1560" alt="Screenshot 2023-08-08 at 12 20 02 PM" src="https://github.com/google/fonts/assets/5162664/ad5772ee-fcbe-460e-9ff7-cc68c4865666">

I am not the author, and this has an RFN, but this seems like a nice project with a great backstory. It might be worth looking into removing the RFN.

**Comment by inferno986return** [comment]
This is a good find! The typefaces are familiar, yet distinctive at the same time. Hopefully, this gets approved.

**Comment by yanone** [comment]
Added a comment to the existing issue in the upstream repo about adding it to GF, which is 2 years old and has never received a reply from the author yet.

It's unclear if and how we proceed without the author's consent, whether we want to fork it or not, which we will probably have to talk about.

**Comment by chrissimpkins** [comment]
Thanks Yanone.  We'll wait for the author to complete their updates and plan to onboard this with those changes later in the year.  @adrianzwz please let us know when your work is complete. Ty!

**Comment by chrissimpkins** [comment]
Tentatively pushing back to Q3 2025

**Comment by adrianzwz** [comment]
Hi @chrissimpkins , I would like to inform that the updates are mostly completed!
The repository is now transferred to an organization: https://github.com/localremotetw/TASA-Typeface-Collection

I've tried following along the GF-guides, and updated the repo structure too.
Though not entirely sure how actions work, but the font generation seems to be working fine for now!

**Comment by emmamarichal** [comment]
Hi @adrianzwz!
Excellent, thank you very much. I'll have a look quickly. (The team evolved a bit, I'll probably be in charge of that project)

**Comment by muuvmuuv** [comment]
Yeah love that! Waited so long to start my Aero business haha

**Comment by emmamarichal** [comment]
@adrianzwz 

I exported the fonts without any issues to start a review. I rarely see fonts that are this clean and well executed. Congratulations, and again, thank you for this high-quality submission!

I'm just waiting on a confirmation from my team regarding the name, as it's the only fail we encountered:
```
FAIL Check family name for GF Guide compliance. (googlefonts/name/familynamecompliance)
🔥 FAIL "TASA Orbiter" contains an abbreviation. [code: abbreviation]
```

But I believe making an exception won't be a problem.

> Also, could you upload the exported fonts either directly to the repo or in a release? We need them to package the fonts and add them to the Google repo (we can’t access the fonts built by the GitHub Actions directly). 
> (If it's in a release, please include the OFL.txt in the folder). 

Edit: oh sorry, I just see you actually have a release! my bad

I’ll get in touch via email regarding the article and designer profiles.
Thanks again!

**Comment by adrianzwz** [comment]
@emmamarichal 

Thank you for your response!
We have received the email and will get back soon.

Also I just discovered some kerning inconsistencies so might push a small patch in a few days.
Is it still possible to update the fonts before onboarding?
If so I'll notify here once updated.

Thanks!

**Comment by emmamarichal** [comment]
@adrianzwz 
Yes of course, I didn't start the PR, so you can still add some modifications. Let me know here or by email once it's ready :)
thank you very much!

**Comment by adrianzwz** [comment]
I just found an issue of the **black circled figures**
The fonts export normally with Glyphs, but the path directions are wrong when the fonts are generated with GitHub Actions

currently I don't know how to fix them other than decomposing the components in the source files

<img width="1221" height="402" alt="Image" src="https://github.com/user-attachments/assets/c1f7b8a0-8429-444c-9879-b7de359ff423" />

<img width="763" height="574" alt="Image" src="https://github.com/user-attachments/assets/cb3083b4-ac5d-412a-b3a4-6bc2509055ef" />

**Comment by adrianzwz** [comment]
@emmamarichal 
I had the **black circled figures** decomposed for now as a workaround
and pushed a patch in v2.011 — please use this version

sorry for the frequent small updates!

**Comment by emmamarichal** [comment]
@adrianzwz, no problem! thank you for the update, I'll make the change :)
